assets/js/googlesitekit/datastore/forms/index.test.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * core/forms data store tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	createTestRegistry,
+	unsubscribeFromAll,
+} from 'tests/js/utils';
+import { STORE_NAME } from './constants';
+
+describe( 'core/forms store', () => {
+	const formName = 'Form 1';
```

Comment:
We should move these variables down into each test's code. Them being here avoids repetition, but it makes the test code much harder to read/reason about because the variables we use as input to the tests aren't scoped to a particular test nor near them.

It's helpful to approach test code differently than the rest of our code: here we want a hyper-focus on readability and context, even if that means repeating things across many tests 😄